xml/cl.xml

```diff
@@ -5718,6 +5736,28 @@ server's OpenCL/api-docs repository.
             <require comment="cl_channel_type">
                 <enum name="CL_UNORM_INT16"/>
                 <enum name="CL_FLOAT"/>
+        </extension>
```

Comment:
There is a missing XML tag before this line (`</require>`).

Reply:
Done.